src/python/pants/backend/python/goals/export.py

```diff
@@ -50,6 +52,16 @@ def debug_hint(self) -> str | None:
         return self.resolve
 
 
+@dataclass(frozen=True)
+class _ExportIsortRequest:
+    pass
+
+
+@dataclass(frozen=True)
+class _ExportDocformatterRequest:
```

Comment:
Hit `Too many sources of dependency` errors when trying to use the same `_ExportToolRequest` for both `isort` and `docformatter` - this was the easiest fix to get the rest of the code hammered out